sdk/personalizer/azure-ai-personalizer/setup.py

```diff
@@ -0,0 +1,78 @@
+from setuptools import setup, find_packages
+import os
+from io import open
+import re
+
+# example setup.py Feel free to copy the entire "azure-template" folder into a package folder named
+# with "azure-<yourpackagename>". Ensure that the below arguments to setup() are updated to reflect
+# your package.
+
+# this setup.py is set up in a specific way to keep the azure* and azure-mgmt-* namespaces WORKING all the way
+# up from python 2.7. Reference here: https://github.com/Azure/azure-sdk-for-python/wiki/Azure-packaging
+
+PACKAGE_NAME = "azure-ai-personalizer"
+PACKAGE_PPRINT_NAME = "Azure Personalizer"
```

Comment:
```suggestion
PACKAGE_PPRINT_NAME = "Personalizer"
```

Otherwise the title on PyPi will be Microsoft Azure Azure Personalizer Client Library for Python

Reply:
Fixed.